packages/moonstone/Marquee/Marquee.js

```diff
@@ -2,7 +2,7 @@
  * Provides components for displaying and controlling marqueed text.
  *
  * @example
- * <MarqueeText>Lorem ipsum dolor sit amet</MarqueeText>
+ * <Marquee>Lorem ipsum dolor sit amet</Marquee>
```

Comment:
Last one: Can you make this string longer and set `marqueeOn="render"` so that it can marquee?  It might be good to copy the docs from `ui/MarqueeDecorator` into `moonstone/Marquee` so that the user can see what can be set.